VideoAPI/Video.API/Controllers/ConsultationController.cs

```diff
@@ -228,19 +230,51 @@ public async Task<IActionResult> StartPrivateConsultationWithEndpointAsync(Endpo
                 _logger.LogWarning(message);
                 return Unauthorized(message);
             }
-            
+
             if (!endpoint.DefenceAdvocate.Trim().Equals(defenceAdvocate.Username.Trim(), StringComparison.CurrentCultureIgnoreCase))
             {
                 var message = "Defence advocate is not allowed to speak to requested endpoint";
                 _logger.LogWarning(message);
                 return Unauthorized(message);
             }
-            
+
             await _videoPlatformService.StartEndpointPrivateConsultationAsync(conference, endpoint, defenceAdvocate);
 
             return Accepted();
         }
-        
+
+        /// <summary>
+        /// End Judge/JoH consultation
+        /// </summary>
+        /// <param name="request">EndConsultationRequest</param>
+        /// <returns>Ok</returns>
+        [HttpPost("end/{roomid}")]
+        [SwaggerOperation(OperationId = "EndJudgeJohConsultationRequest")]
```

Comment:
The operation id should not be limited to judge/joh since I believe this will be used for all types of consultation

Reply:
Initially I wanted to keep this separate from normal consultations as the start and end was quite different. But I guess we can re-use this for Participants now.

Reply:
Updated names